api-report/container-definitions.api.md

```diff
@@ -167,6 +168,8 @@ export interface IContainerContext extends IDisposable {
     readonly connected: boolean;
     // (undocumented)
     readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
+    // (undocumented)
+    readonly disposeFn: (error?: ICriticalContainerError) => void;
```

Comment:
I would prefer this is explicitly marked as optional, so Runtime code knows to check if it's present before calling it (as you do)